app/src/main/res/values/strings.xml

```diff
@@ -53,10 +53,10 @@
     <string name="nav_menu_graphs">Graphs</string>
     <string name="nav_menu_settings">Settings</string>
     <string name="nav_feedback">Feedback</string>
-    <string name="nav_menu_bitcoin_donation">Donate Bitcoin</string>
+    <string name="nav_menu_bitcoin_donation">Donate bitcoin</string>
```

Comment:
Could you please update localization by weblate?
For this specific translation - Bitcon is in general name of a crypto currency. By my opinion, there should be capital letter.

Reply:
"Bitcoin" is the organization, "bitcoin" is the currency. Though it is voluntary, that is the distinction i have found.